pkgs/development/tools/sqsh/default.nix

```diff
@@ -17,11 +17,13 @@ in stdenv.mkDerivation rec {
 
     substituteInPlace src/cmd_connect.c \
       --replace CS_TDS_80 CS_TDS_73
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+    substituteInPlace configure --replace "libct.so" "libct.dylib"
   '';
 
   enableParallelBuilding = true;
 
-  buildInputs = [ freetds readline ];
+  buildInputs = [ freetds readline ] ++ stdenv.lib.optional stdenv.isDarwin libiconv;
```

Comment:
We don't want to use conditionals here. It will be null on Glibc systems anyway.

Reply:
Done.